metadata.xhtml

```diff
@@ -174,6 +174,22 @@
 <!-- /wp:table -->
 
 <!-- wp:heading {"level":4} -->
+<h4>Colorimetry classification scheme</h4>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p>This CS lists the colorimetry (colour space) options that can be applied to video content.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Latest version:&nbsp;<a href="/metadata/cs/ColorimetryCS.xml">permalink</a></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:table {"className":"table table-sm"} -->
+<figure class="wp-block-table table table-sm"><table><thead><tr><th>URI</th><th>DVB Standard(s)</th><th>Description</th></tr></thead><tbody><tr><td><a href="/metadata/cs/2020/ColorimetryCS.xml">urn:dvb:metadata:cs:ColorimetryCS:2020</a></td><td>A177</td><td>Initial version</td></tr></tbody></table></figure>

```

Comment:
Suggesting, instead of just "A177", to be more specific as in "DVB BlueBook A177r3".

Reply:
- on first bullet - Column title is "DVB Standard" already. It is also likely that it will exists in A177r4 and future versions so I would keen it versionless unless a new CS comes into effect in the future (then we can add A177 versions for 2020).
- on second bullet - we can do this if ETSI standard takes precedence over DVB

Reply:
> * on first bullet - Column title is "DVB Standard" already. It is also likely that it will exists in A177r4 and future versions so I would keen it versionless unless a new CS comes into effect in the future (then we can add A177 versions for 2020).

I think it would be helpful to capture on the website with which version of the spec each version of the CS (corresponding to a namespace) was introduced. For the sake of example, let's assume you'd be making r4 of the document, but no changes to the CS. Then no change to the website would be needed. Assuming you'd make chanegs to the CS in r5 of the document, then the table would get a new row, listing the new namespace, and r5 of the document. That way, it'd be easy to get a quick overview of what you need to implement to cover version N of the spec.

> * on second bullet - we can do this if ETSI standard takes precedence over DVB

Great; my intent was to have the Bugzilla issue as a reminder, so it won't be forgotten. That's all.

Reply:
> I think it would be helpful to capture on the website with which version of the spec each version of the CS (corresponding to a namespace) was introduced. 

Done
 
> Great; my intent was to have the Bugzilla issue as a reminder, so it won't be forgotten. That's all.

Wont need a Bugzilla to remind us of 'normal working procedure'